### PR TITLE
Fix attachment dl on maintenance fixes #5099

### LINKED ISSUE #5099: Download of attachment when forum is in maintenance create many erros
#### Description
arround 87 error get created when a not admin try to access a attachment,
while the forum is in maintenance mode.

### Steps to reproduce
1.  Create a topic with attachment
2.  Copy the url to the download, should be something like that index.php?action=dlattach;topic=123404.0;attach=5
3.  go in maintenace mode
4.  call the url with a non admin user

### Environment (complete as necessary)
- Version/Git revision: newest
- Database Type:
- Database Version:
- PHP Version:

### Additional information/references
Related sm.org https://www.simplemachines.org/community/index.php?topic=562960.0

Ther is an optimization in place that:
https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/index.php#L184-L189
that the loadTheme is not called when ``$_REQUEST['action'] == 'dlattach'``
which is deadly in maintenance mode -> disable this optimization for this case.

### PR DESCRIPTION
**Pull request by albertlast**
issue: #5099
https://www.simplemachines.org/community/index.php?topic=562960.0

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -221,7 +221,7 @@ function smf_main()
 			require_once($sourcedir . '/LogInOut.php');
 			return ($_REQUEST['action'] == 'login2' ? 'Login2' : ($_REQUEST['action'] == 'logintfa' ? 'LoginTFA' : 'Logout'));
 		}
-		// Don't even try it, sonny.
+		// Don't even try it, sorry.
```
Comment (live627) [inline index.php]: !!!

Comment (albertlast) [reply index.php]: want you your sonny back?

Comment (live627) [reply index.php]: yes, please. I didn't write it, but it seems quite humorous.

...maybe that comment only makes senses in English-speaking countries?

Comment (live627) [reply index.php]: > Don't even try it, sorry

That seems like broken English to me.

Comment (albertlast) [reply index.php]: When you look into 2.0 code you'll find sonny ther, too.

**Comment by live627** [comment]
> Confirmed: that fixes the problem, thanks

https://www.simplemachines.org/community/index.php?topic=562960.msg3994072#msg3994072

**Review by live627 — APPROVED** [review]

**Pull request merged**